### DISCUSSION TITLE
synchronize two NPM registries

### DISCUSSION THREAD

**Discussion by wangxuanqi** [verdaccio/verdaccio#2355] [Q&A] [answered]
**For project requirements, two local private NPM registries are run through two local ports. Both NPM registries have the same module but different versions. I want to synchronize their versions not through "NPM publish", but through nodejs or verdaccio API.  Any suggestions?**

**Thanks!**

**Comment by juanpicado** [comment, marked as answer]
seems https://github.com/verdaccio/verdaccio/discussions/2306
